content/en/dashboards/sharing.md

```diff
@@ -19,7 +19,39 @@ further_reading:
 
 ## Overview
 
-Shared graphs and screenboards allow you to display metric, trace, and log visualizations outside of Datadog.
+Shared dashboards and graphs allow you to display metric, trace, and log visualizations outside of Datadog.
+
+## Dashboards
+
+**Share an entire dashboard publicly by generating a URL**:
+1. On the dashboard's page, click the settings cog in the upper right.
+2. Choose the **Generate public URL** option.
+3. Choose configuration options under "Time & Variable Setings" for "Allow changing timeframe" and/or template variables visible tags.
+4. Copy the URL and click **Done**.
+
+**Share with individual emails**: 
```

Comment:
```suggestion
### Send a dashboard link by email

 To share a dashboard with individuals by email:
 
```

Reply:
We're not just sending an email, they are specifically being authorized individually

Reply:
Got it. Updated to be more clear about that.